seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/contract/BaseContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.seonoh.seonohapp.contract
+
+import io.reactivex.disposables.CompositeDisposable
+
+interface BaseContract {
+
+    interface View<T>{
+        val presenter : T
```

Comment:
변수와 타입은 콜론이 변수에 붙어 있어야 하는 것 같아요.
```suggestion
        val presenter: T
```

자동 정렬 기능을 사용하시면 좋을 것 같네요.

Reply:
감사합니다!! 
15f76adf289be5b7aaab2f952161c689af685356